src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -707,35 +704,34 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
 
         const fragmentClasses = [
             [
-                n_fixed_rows && n_fixed_columns ? 'dash-fixed-row dash-fixed-column' : '',
-                n_fixed_rows ? 'dash-fixed-row' : ''
+                fixed_rows && fixed_columns ? 'dash-fixed-row dash-fixed-column' : '',
```

Comment:
as a bool "do we have any fixed rows" wouldn't you need `fixed_rows.headers`?

Reply:
Yes! Missed that one! Updating.

Reply:
OIC you did this with `getFixedRows` - that makes sense. But then it might have been clearer, since this one is for internal use only, to leave it called `n_fixed_rows`...

Reply:
No! Because I didn't want to change all the table's logic for what only amounts to an API change, the input props get processed early and transformed back into a number to work wih the previous code. In the Dash realm this is `{ headers: bool, data: number }` but in the TS realm this is still a `number`.

Initial transform: https://github.com/plotly/dash-table/pull/446/files#diff-2fcfaf68a2e368e80fa272945bdce51aR47

So by the time this gets fed to the controlled table, these props are respecting this props interface: https://github.com/plotly/dash-table/compare/core-issue-58-refactor-api...issue58-content-style-exp#diff-79448babc2419344d331d4d4f127f315R364

Reply:
Got it. Would just be easier to read if `fixed_rows` always meant the object `{headers, data}` and when you transform it to a number it's still called `n_fixed_rows`